EXP-Client/todo/Todo.js

```diff
@@ -0,0 +1,80 @@
+import * as React from 'react';
+import { Provider } from 'react-native-paper';
+import {
+  StyleSheet,
+  Text,
+  View,
+} from 'react-native';
+import ProgressCircle from 'react-native-progress/Circle';
+import AddTaskMenu from './AddTaskMenu';
+import TaskList from './TaskList';
+
+function Graph({ tasks }) {
+  const completedCount = 2; // tasks.reduce(({ name, deadline, completed }, count) => completed ? count + 1 : count, 0);
```

Comment:
Why are we not going ahead with the commented technique?

Reply:
Doesn't work currently because it's not hooked up to the backend